GothicLlamasAPI/src/main/java/com/kainos/ea/objects/JobRolesAndSpecificationMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.kainos.ea.objects;
+
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class JobRolesAndSpecificationMapper implements RowMapper<CapabilitiesAndRoles> {
+    @Override
+    public CapabilitiesAndRoles map(ResultSet r, StatementContext ctx) throws SQLException {
+        CapabilitiesAndRoles cap = new CapabilitiesAndRoles();
+        cap.setJobRole(r.getString("Job Role"));
+        cap.setJobDescription(r.getString("Job description"));
+        cap.setLinkToFullDescription(r.getString("Link to SharePoint"));
```

Comment:
I see some inconsistency with the column names. `Job Role` is in title case, `Job description` is in sentence case, and `Link to SharePoint` could be either (if SharePoint is a proper noun). Also in `JobRoleMapper` you just have `jobRole` – camel case.

You might want to extract common column names into constants at some point, but you won't be able to if they're all slightly different like this. It also makes it harder to guess in future work exactly what they are without looking at the SQL.